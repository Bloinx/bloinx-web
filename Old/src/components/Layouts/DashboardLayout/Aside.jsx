import React from "react";
import PropTypes from "prop-types";
import styles from "./Aside.module.scss";

export default function Aside({ children }) {
  return <div className={styles.Aside}>{children}</div>;
}

Aside.propTypes = {
  children: PropTypes.node.isRequired,
};
