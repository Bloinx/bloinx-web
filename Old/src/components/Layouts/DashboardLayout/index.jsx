import React from "react";
import PropTypes from "prop-types";
import styles from "./index.module.scss";

export default function DashboardLayout({ children }) {
  return <div className={styles.DashboardLayout}>{children}</div>;
}

DashboardLayout.propTypes = {
  children: PropTypes.node.isRequired,
};
