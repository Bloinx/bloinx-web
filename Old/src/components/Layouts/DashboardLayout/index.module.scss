.DashboardLayout {
  margin: 0px;
  padding: 0px;
  border: 1px solid red;
  display: flex;
  flex-direction: row;
}
