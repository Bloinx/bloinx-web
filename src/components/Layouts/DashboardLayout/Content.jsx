import React from "react";
import PropTypes from "prop-types";
import styles from "./Content.module.scss";

export default function Content({ children }) {
  return <div className={styles.Content}>{children}</div>;
}

Content.propTypes = {
  children: PropTypes.node.isRequired,
};
