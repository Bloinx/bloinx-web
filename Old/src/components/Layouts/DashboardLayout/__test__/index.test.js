import React from 'react';
import renderer from 'react-test-renderer';

import DashboardLayout from '../index';

describe('DashboardLayout', () => {
  it('renders correctly', () => {
    const Foo = () => <div />;
    const tree = renderer.create(
      <DashboardLayout><Foo /></DashboardLayout>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
