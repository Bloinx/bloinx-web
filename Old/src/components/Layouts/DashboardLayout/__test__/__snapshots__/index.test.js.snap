// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardLayout renders correctly 1`] = `
<div
  className="DashboardLayout"
>
  <div />
</div>
`;
